test/xprof_vm_info_tests.erl

```diff
@@ -52,4 +52,67 @@ get_available_funs_test_() ->
               ?assertEqual([[?M, get_available_funs, 1]], L2),
               L3 = ?M:get_available_funs(<<"xprof_vm_info:get_available_funs/99">>),
               ?assertEqual([], L3)
-      end}].
+      end},
+     {"Match-spec fun matching",
+      fun() ->
+              L1 = ?M:get_available_funs(<<"xprof_vm_info:get_available_funs(_) ->">>),
+              ?assertEqual([[?M, get_available_funs, 1]], L1),
+              L1 = ?M:get_available_funs(<<"xprof_vm_info:get_available_funs return_trace()">>),
```

Comment:
Ok, totally fine.  If `return_trace` is always there, then it doesn't make sense to introduce the checkbox -  I can't see any other use of it right now.